src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -129,7 +134,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        if (null === $process = $this->createServerProcess($output, $address, $documentRoot, $input->getOption('router'), $env, null)) {
+        if (null === $process = $this->createServerProcess($output, $address, $documentRoot, $router, null)) {
```

Comment:
Agreed, the null there isn't necessary.  I was tempted to remove it but didn't want to meddle with the code any more than I needed to.  @xabbuh - as the author of this class, would you like me to remove it?